ansible_collections/juniper/device/tests/integration/targets/prepare_junos_tests/tasks/prepare.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Debug task
+  ansible.builtin.debug:
+    msg: "START prepare_junos_tests/main.yaml"
+
+- name: Wait for netconf server to come up
+  delegate_to: localhost
+  tags: netconf
+  ansible.builtin.wait_for:
+    host: "{{ hostvars[item].ansible_host }}"
+    port: 830
```

Comment:
Can you load "netconf ssh" as pre-check  validation

Reply:
not supported 
fatal: [junos_test]: FAILED! => {"msg": "to use the 'ssh' connection type with passwords or pkcs11_provider, you must install the sshpass program"}